core/invoker/src/main/scala/whisk/core/invoker/Invoker.scala

```diff
@@ -442,12 +443,25 @@ object Invoker {
             val groupid = "invokers"
             val maxdepth = ContainerPool.getDefaultMaxActive(config)
             val consumer = new KafkaConsumerConnector(config.kafkaHost, groupid, topic, maxdepth)
+            val producer = new KafkaProducerConnector(config.kafkaHost, ec)
             val dispatcher = new Dispatcher(consumer, 500 milliseconds, 2 * maxdepth, actorSystem)
 
-            val invoker = new Invoker(config, instance, dispatcher.activationFeed)
+            val invoker = if (Try(config.invokerUseReactivePool.toBoolean).getOrElse(false)) {
+                new InvokerReactive(config, instance, dispatcher.activationFeed, producer)
+            } else {
+                new Invoker(config, instance, dispatcher.activationFeed, producer)
+            }
+            logger.info(this, s"using $invoker")
+
             dispatcher.addHandler(invoker, true)
             dispatcher.start()
 
+            Scheduler.scheduleWaitAtMost(1.seconds)(() => {
+                producer.send("health", PingMessage(s"invoker$instance")).andThen {
```

Comment:
should you remove the health pings from old `Invoker` for good measure to avoid double pings if the flag is false?